src/debugpy/common/log.py

```diff
@@ -143,12 +163,13 @@ def write(level, text, _to_files=all):
     return text
 
 
-def write_format(level, format_string, *args, **kwargs):
+def write_format(level: LevelTypes, format_string: str, *args: Any, **kwargs: Any):
     # Don't spend cycles doing expensive formatting if we don't have to. Errors are
     # always formatted, so that error() can return the text even if it's not logged.
     if level != "error" and level not in _levels:
         return
 
+    text = ""
```

Comment:
Is it meant as a type assertion, or to silence a non-initialized variable warning? If the latter, it might be clearer to do `raise reraise_exception()` to indicate to the linter that there's no exit out of the `except` block.